axelor-account/src/main/resources/views/Year.xml

```diff
@@ -88,7 +76,7 @@
     	<call class="com.axelor.apps.account.web.YearController" method="adjust"/>
     </action-method>
     
-    <action-validate name="action-year-validate-generate-period">
+    <action-validate id="account-action-year-validate-generate-period" name="action-year-validate-generate-period">
 	    <error message="To generate periods, please fill in the limits of the fiscal year." if="fromDate == null || toDate == null" />
```

Comment:
I think we can delete this and keep the action validate in base, to avoid code duplication.